src/Compilers/Core/MSBuildTask/ManagedCompiler.cs

```diff
@@ -211,6 +224,12 @@ public ITaskItem[] ResponseFiles
             get { return (ITaskItem[])_store[nameof(ResponseFiles)]; }
         }
 
+        public bool SkipCompilerExecution
```

Comment:
I hadn't considered the command line + also really build scenario, makes sense now.